ee/session_recordings/playlist_counters/recordings_that_match_playlist_filters.py

```diff
@@ -363,32 +425,29 @@ def count_recordings_that_match_playlist_filters(playlist_id: int) -> None:
                 "playlist_id": playlist_id,
                 "playlist_short_id": playlist.short_id if playlist else None,
                 "posthog_feature": "session_replay_playlist_counters",
-                "filters": playlist.filters if playlist else None,
-                "query": query_json,
```

Comment:
the log and the prom counter were showing up but these exceptions aren't showing up in posthog... so sending slightly less in case something in these blobs of data was breaking the event